src/scripts/modules/media/tabs/contents/toc/page.coffee

```diff
@@ -53,6 +53,11 @@ define (require) ->
       e.stopPropagation()
 
       $link = $(e.currentTarget)
+      if $link[0].href.indexOf("#") > -1
```

Comment:
Rather than using `$link[0].href` can you pull the value into a local variable? See https://github.com/Connexions/webview/pull/1545#discussion_r120736703